src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -233,7 +261,7 @@ public Path getJarPath(String jarName) {
   /**
    * Checks whether the configured Cloud SDK Path is valid.
    *
-   * @throws AppEngineException when there is a validation error.
+   * @throws AppEngineException when there is a validation error
    */
   public void validate() throws AppEngineException {
```

Comment:
if validate() expects errors why not just return the issues rather than throw them?